server/src/http/routes/catalogue.ts

```diff
@@ -35,20 +34,20 @@ export default () => {
       const qs = req.query
       const query = qs && qs.query ? JSON.parse(qs.query) : {}
 
-      const response = await getFormations(query, 1, 10000)
-      const totalFormations = response.formations.length
+      const formations = await getFormations(query)
```

Comment:
@rimiti I don't see any use of it in the front, can it be removed ?

Reply:
This controller could be remove for a new one that return results directly from the database (not from the catalogue anymore)

Reply:
SearchPage uses `/catalogue/formation/` but `/catalogue/formation/filter` doesn't seem do be used anywhere